lib/hsa/mcwamp_hsa.cpp

```diff
@@ -5007,6 +5018,9 @@ HSABarrier::getEndTimestamp() override {
 }
 
 
+// Activity profiling instantiation
+ACTIVITY_PROF_INSTANCES();
```

Comment:
Should replace usage of this macro, see previous comment on this

Reply:
I see two solutions I wrote in the previous comment on this.